### DISCUSSION TITLE
Custom Web Service Listner

### DISCUSSION THREAD

**Discussion by sahil-kaushik** [nextgenhealthcare/connect#5763] [Q&A]
Hi 

Can someone provide a sample project or a guide to create a Custom Web Service Listener? I tried to compile the jar, but got a lot of unresolved classes. 
Could not find anything descriptive on the search. 

Intent: Create a Web Service Listener to accept IHE XDR payload (with embedded CDA document)

Would really appreciate any help towards the right direction. 

Mirth version 3.9.X

Thanks!

**Comment by mjtrac** [comment]
Perhaps this will help? https://www.linkedin.com/pulse/custom-extension-mirth-connect-claudio-baeza-gonzalez/

**Comment by ourlang** [comment]
@sahil-kaushik Hello, may I ask if you are available now Custom Web Service Listener？Would really appreciate any help towards the right direction.

Mirth version 4.4.X   jdk  17.0.8
